### DISCUSSION TITLE
Saving the modified card to make changes permament?

### DISCUSSION THREAD

**Discussion by wooshaq** [Prolo1/Character-Morpher#22] [General]
Sorry, i'm not sure how this plugin works but I would like to apple changes and save the card to keep the modified body but upon load and disabling the plugin I get the original body, what option should I chose for it to work? I managed to save the changes like twice but I'm not sure what I did.

**Comment by Prolo1** [comment]
for that functionality, all you need to do is disable the "save ext. data" toggle and the the card will save like you expect. that toggle allows people to save the card w/o changing the original body so it will look the same when loaded the next time.

**Comment by Prolo1** [comment]
The mod has a lot of toggles (some of which I want to purge or make more intuitive), but they do allow for a lot of customization which is what I was aiming for. 

Feel free to open this reply again if this doesn't completely answer your question.

**Comment by Prolo1** [comment]
actually I think ill move this to discussions instead since it makes more sense for it to be there

**Comment by wooshaq** [comment]
Nice. Ill give it a try. What about your clothing changing mod? Im lookingbto make clothes chenge easily on sunshinw studio, are you planning to release it on koikatsu?

**Reply by Prolo1** [reply]
Sorry for the late reply (I didn't get a ping like I normally do). I can't remember why I didn't do it before but when I make a build of it I'll add another reply with the link.

**Reply by wooshaq** [reply]
Great, what i could recommend it perhaps an option to keep original head proprtion as changing body is ok but it would be recommended to keep old head propotion since i had to set the haed value to 0 on every new run since the value alwayscdefaults to 100 on every editor run. Some character just look unnatural with new head. What abbout nipples and areolas? Could them be chwitched as well? Also would it be possible to save modification made within the studio as char save is only possible within the koikatsu editor, or am i wrong? Your plugin is god send nevertheless.

**Reply by Prolo1** [reply]
Sup. I'll answer the questions you asked as best as possible.
- the idea of having an option to change the default face/nipple options to zero is nice and all (and I should probably change it to 50), but the same argument could be said about every single slider in this mod and I have A LOT of them. A slightly better solution would be to add an option to use your default slot as a template for other slots (which I'll do, and it's actually easier).

-  character saves can only be made in the main game. However, if you make changes to a character in a scene with my mod and save the scene all the changes are saved for the characters in the scene. The scene basically combines all the cards/objects together into a jumbo card with blackjack and hookers so I think it's already doing what's intended. If you want to extract a card from a scene there is a mod that can do that, I just can't remember what it is.

Hope that answers everything.

**Reply by wooshaq** [reply]
Ok, fair enough, perhaps you could at least add an option for mod to remember set values ifvthus is enen possible so if i set head value to 0 it would alwayscstay that way on load but this is only minor complaint. I thought i tried to save an edited studio scene and it remained unchanged upon load but ill experiment some more once ill get back from my holiday.

**Reply by Prolo1** [reply]
I'm just making sure but are you saving the slots as you change them? that is something you have to do manually. I could also add an option to auto-save on change but I ran into issues doing that once and kinda just forgot about it.

**Reply by wooshaq** [reply]
I guess not, i only loaded the new body amd save the scene. What else do i need to do?

**Reply by Prolo1** [reply]
at the bottom of the character morpher menu, there is a save button that allows you to save the slider values you make to the current slot. making a new slot will not auto override your slider setting since there have been cases where people (AKA. me) have made a new slot, forgot to save their old slot and lost their progress. If you load the slot after you create it, it will use show the default values.

**Reply by wooshaq** [reply]
Ill see when ill get back. Seems complicated tbh as for me i prepared a lot of cards showing proportions so it is easy to know what i load. Is it not possible to save loaded body into the scene without the slot think. Perhaps im too bias for now as i never tried your solution but saving cards within game seems much easier, nevertheless thank you for your mods.
